libgearman/protocol/submit.h

```diff
@@ -48,13 +48,6 @@ gearman_return_t submit(gearman_universal_st&,
                         const gearman_string_t &function,
                         const gearman_string_t &workload);
 
-gearman_return_t submit_background(gearman_universal_st&,
```

Comment:
This is here for API compatibility. While it is undocumented, and seemingly only used by internal things, it's in the public symbols and I don't think it's worth passing a static analyzer to break somebody who accidentlly used it.